sdk/core/core-client-rest/src/operationOptionHelpers.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { OperationOptions, RequestParameters } from "./common";
+
+export function operationOptionsToRequestParameters(options: OperationOptions): RequestParameters {
+  return {
+    allowInsecureConnection: options.requestOptions?.allowInsecureConnection,
+    timeout: options.requestOptions?.timeout,
+    skipUrlEncoding: options.requestOptions?.skipUrlEncoding,
+    abortSignal: options.abortSignal,
+    onUploadProgress: options.requestOptions?.onUploadProgress,
+    onDownloadProgress: options.requestOptions?.onDownloadProgress,
+    tracingOptions: options.tracingOptions,
+    headers: { ...options.requestOptions?.headers },
+    onResponse: options.onResponse,
+  };
```

Comment:
I haven't add a unit test for it now, because I think the logic is pretty straightforward. but let me know how you think @xirzec Thanks.